target/linux/ipq40xx/image/Makefile

```diff
@@ -164,6 +164,15 @@ define Device/avm_fritzrepeater-3000
 endef
 TARGET_DEVICES += avm_fritzrepeater-3000
 
+define Device/cilab_meshpoint-one
+	$(call Device/8dev_jalapeno)
```

Comment:
As stated earlier, please avoid including another device definition and create a common definition for jalapeno board (following the same logic why we have a DTSI and not derive one DTS from the other).

Reply:
Sure, will do that.